### ISSUE TITLE
Read receipts are noisy for screen readers

### ISSUE THREAD

**Issue by ndarilek** [vector-im/riot-web#2716]
I assume these are the read receipts, anyway. My screen reader presents a bunch of alt attributes containing Matrix IDs throughout the message flow. This makes reading individual messages, or continuously reading the logs, a noisy proposition.

I'd suggest adding an `aria-hidden="true"` on the element containing the read receipt position in the chatroom, then exposing this information via some other means. I'm probably more interested in looking up how far an individual user has read in a chatroom, but I don't know if there are room-specific detail pages for members. But in any case, hiding these from screen readers would be a good first step.

Maybe there can be a panel that displays a table, showing users and their position in the room? I noticed that, when I click on the overflow row of icons on Android, I seem to get a list like this. A table sorted from least to most active would be helpful.

Thanks!

**Comment by ara4n** [comment]
I've hidden RRs by default in matrix-org/matrix-react-sdk@d0d056e.  Will leave this open to work out a better way to show them...

**Comment by pvagner** [comment]
@ndarilek I think this is now addressed partially after @JaniM has turned most of the divs into accessible buttons in JaniM/riot-web@5edb5f6487404b89da45f13f3d8444dd1f533d0f - now also merged into develop and deployed on riot.im/develop.
At least for me with Firefox on Linux with Orca I don't have these receipts spoken when reading by line, by word or by letter. However I can tab / shift+tab over the buttons corresponding with individual receipts and activate to show member details screen.
When focusing and activating these buttons with the enter or a space bar press perhaps something happens however I fail to identify that action and I can't work out if this is an issue or not.
Also if there are more than 5 receipts associated with a message + sign is not focusable / activatable so full list can be expanded with the keyboard alone.
As I have said the verbosity issue is addressed for me. Need to verify with other browser vs AT combos whenever possible.

**Comment by lukebarnard1** [comment]
Having tested it, I observe that clicking, pressing space and pressing enter all do the same thing now, which is to expand and contract the list.
